FirebaseMessaging/Apps/Sample/Sample/ContentView.swift

```diff
@@ -221,7 +221,7 @@ struct ContentView: View {
   }
 
   func deleteFCM() {
-    Messaging.messaging().delete { error in
+    Messaging.messaging().deleteData { error in
```

Comment:
@ryanwilson After renaming, the swift API calls it deleteData, does that look good to you? or I should rename swift API to deleteMessagingData as well?

Reply:
This looks good to me - thanks!